pages/blog/2023-summary.md

```diff
@@ -0,0 +1,164 @@
+
+<Figure
+  src="/img/posts/2023-summary/referred-by-Twitter.webp"
+/> 
+
+Although the decline is shallow, I'm not extremely worried. Most top tweets for this year were appreciation tweets balanced up with technical tweets. Additionally, we don't use any paid services, and having any additional investment in this platform is not worth it, at least for now.
+
+#### YouTube
+
+The AsyncAPI YouTube channel performed way better than expected, especially as we moved from online to in-person conferences. We currently have over 364 people subscribed to our channel, increasing total subscribers to over 1,46k. Our live streams and conference recordings bring in a lot of views, and I'm optimistic about 2024 as well.
+
+<Figure
+  src="/img/posts/2023-summary/youtube.webp"
+/> 
+
+#### Newsletter
+On the 5th of April, 2023, we sent out our first AsyncAPI Community Updates Newsletter. The main aim was to summarize monthly activities and initiatives in the community straight to your inbox.
+Over the past nine months, the December Edition was the most popular. This was due to the release of the AsyncAPI v3, which was one of our major accomplishments in 2023.
+
+<Figure
+  src="/img/posts/2023-summary/total-opens.webp"
+/> 
+
+And, of course, I'm not exempt from unsubscribers. Although the number might seem minimal to some, it still stings as a writer.
+
+<Figure
+  src="/img/posts/2023-summary/unsubscribed.webp"
+/> 
+
+Looking forward to sending the 2024 Editions!!🎉🎉
+
+### Google Analytics
+
+The pageviews on the AsyncAPI website increased significantly over the past year. Our major contributor to these metrics comes from docs, and we expect to see more in the coming year as people want to know more about the AsyncAPI v3.
+
+<Figure
+  src="/img/posts/2023-summary/google-analytics.webp"
+/> 
+
+We also lost 21 days' worth of data during October due to a PR update that affected the analytics workflow, but we are doing well.
+
+### Google Search Console
+
+Google Search Console is one of my favorite tools as it shows how much impact and interest we are gaining. Compared to 2022, we can see a surge in impressions and, slowly but surely, in clicks as well. 
+
+<Figure
+  src="/img/posts/2023-summary/google-console.webp"
+/> 
+
+### Github
+
+The number of contributors in the community keeps growing, from having over **1600** contributors in 2022 to **2320** contributors in 2023. 
+
+<Figure
+  src="/img/posts/2023-summary/Total-Contributors.webp"
+/> 
+
+Although we have a handful of maintainers, we try our best to manage, resolve, and close issues as much as we can. Additionally, we are focusing on growing the number of maintainers in the coming years.
+
+<Figure
+  src="/img/posts/2023-summary/Issue-History.webp"
+/> 
+
+### NPM
+
+Numbers don't lie, and yes, these are the total downloads of the top AsyncAPI packages hosted in the Node Package Manager. We can also see the adoption of AsyncAPI spec downloads increased from 5M to 17M since 2022.
```

Comment:
```suggestion
Numbers don't lie; yes, these are the total downloads of the top AsyncAPI packages hosted in Node Package Manager. We can also see the adoption of AsyncAPI spec downloads increased from 5M to 17M since 2022.
```